src/nrniv/Makefile.am

```diff
@@ -87,7 +87,12 @@ libnrniv_la_SOURCES = nrnmenu.cpp shape.cpp classreg.cpp rotate3d.cpp \
 	$(PARSRC1) bgpmeminfo.c \
 	netpar.cpp partrans.cpp splitcell.cpp multisplit.cpp \
 	bbsavestate.cpp nrnbbcore_write.cpp pysecname2sec.cpp \
-	nrndae.cpp matrixmap.cpp geometry3d.cpp
+	nrndae.cpp matrixmap.cpp geometry3d.cpp \
+    nrnbbcore_write/callbacks/bbcore_callbacks.cpp \
+    nrnbbcore_write/data/cell_group.cpp \
+    nrnbbcore_write/data/datum_indices.cpp \
+    nrnbbcore_write/io/bbcore_write.cpp \
+    nrnbbcore_write/utils/corenrn_utils.cpp
```

Comment:
`nitpick` : could you check indentation?  (or this is because of mixed tab / spaces from old files)

Reply:
It is hard to compare even now. Another step after verifying this step would be to reformat.

Reply:
I though I'd fixed this :/  We'll run reformatting with the move to C++